cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -134,6 +134,15 @@ for k,v in yaml.load(sys.stdin).iteritems():
 ''' < "${kube_env_yaml}")
 fi
 
+# Wait for the default service account
+token_found=""
+while [ -z "${token_made}" ]; do
+  sleep .5
+  token_found=$(${KUBECTL} get serviceaccount default -o template -t "{{with index .secrets 0}}{{.name}}{{end}}" || echo "")
```

Comment:
Perhaps I'm being dumb, but what is the echo for here?  If it's to silence the output of kubectl I prefer "2>&1 > /dev/null".  If it's to ensure returning true, then I prefer "|| true".